snippets/American-to-English-Date.md

```diff
@@ -0,0 +1,7 @@
+### American to english date
+Take an American date and split the string re-arranging into english format.
+```js
+var americanDate = "5/10/2004";
```

Comment:
These are globally scoped and not a function. It also is not very adaptable.

Using the date object and a function you can also get some error checking should something go wrong(return undefined or adopt other methods such as allowing to bubble, up to choice)

Possible replacement:

```js
const toEnglishDate = (time) => {try{return new Date(time).toISOString().split('T')[0].replace(/-/g, '/')}catch(e){return}}
```

Reply:
I second this, more of a general-case, should be more useful this way. @Hammie217 you can update your PR if you agree, so we can merge this.

Reply:
For a version that the developer must handle the errors of themselves(fails on invalid date, not a Maybe):

```js
const toEnglishDate = (time) => new Date(time).toISOString().split('T')[0].replace(/-/g, '/')
```